docs/reference/commandline/create.md

```diff
@@ -126,7 +126,8 @@ Options:
   -v, --volume value                Bind mount a volume (default []). The format
                                     is `[host-src:]container-dest[:<options>]`.
                                     The comma-delimited `options` are [rw|ro],
-                                    [z|Z], [[r]shared|[r]slave|[r]private], and
+                                    [z|Z], [[r]shared|[r]slave|[r]private],
+                                    [delegated|cached|consistent], and
```

Comment:
`default` is not in this list because in the short `-v` form, `:default` is not very explanatory and simply omitting it is the same. `default` is in the long `--mount` form as it is an actual consistency mode that can be selected and may be indicated for maximum explanatory effect.

`delegated` is the most relaxed and lowest priority (except for `default`, not present in this form). `cached` is the next most relaxed and higher priority. `consistent` is not relaxed and highest priority.